qemu/tests/cfg/multi_nics_hotplug.cfg

```diff
@@ -24,15 +24,25 @@
             extra_params += "-net none"
         - nic_e1000:
             no ppc64 ppc64le
+            no RHEL.8
+            no Host_RHEL.m8
             pci_model = e1000
             match_string = "Gigabit Ethernet Controller"
             nics = ""
             extra_params += "-net none"
+        - nic_e1000e:
+            no ppc64 ppc64le
+            pci_model = e1000e
+            match_string = "Intel 82574L GbE Controller"
```

Comment:
Anywhere used this string or any other special purpose? I didn't catch it in nic_hotplug.py.
If it's redundant, please remove it.